ci-bench/src/main.rs

```diff
@@ -183,6 +189,70 @@ fn main() -> anyhow::Result<()> {
             mem::forget(stdin);
             mem::forget(stdout);
         }
+        Command::Walltime {
+            iterations_per_scenario,
+        } => {
+            let mut timings = vec![Vec::with_capacity(iterations_per_scenario); benchmarks.len()];
+            for _ in 0..iterations_per_scenario {
+                for (i, bench) in benchmarks.iter().enumerate() {
+                    let start = Instant::now();
+
+                    let resumption_kind = black_box(bench.kind.resumption_kind());
+                    let params = black_box(&bench.params);
+
+                    let (mut client_writer, mut server_reader) = async_io::async_pipe(262144);
+                    let (mut server_writer, mut client_reader) = async_io::async_pipe(262144);
```

Comment:
Is this constant value meaningful?

It gets duplicated in a few places (L203, L204, L206, L222). Maybe worth hoisting into a `const`?

Reply:
I took the constant over from the original benchmarks (`rustls/examples/internal/bench.rs`) and never really thought too much about it. I don't think there is a specific meaning to it, but I'll pull it into a `DEFAULT_BUFFER_SIZE` const.